index.html

```diff
@@ -0,0 +1,68 @@
+<html>
+    <head>
+        <title>Youtube</title>
+        <link rel="stylesheet" type="text/css" href="style.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+    <body>
+        <nav>
+            <div>
+                <image src="https://upload.wikimedia.org/wikipedia/commons/e/e1/Logo_of_YouTube_%282015-2017%29.svg" height= "100px" width="100px">
+            </div>
+
+            <div>
+                <input type="text" name="search" placeholder="search">
+            </div>
+            <div>
+                <button type="button" ><i class="fa fa-search"></i></button>
+            </div>
+
+            <div id="add-video-playlist">
+                <i class="fa fa-file-video-o" aria-hidden="true" style="font-size:24px"></i>
+            </div>
+
+            <div id="bell">
+                <i class="fa fa-bell" style="font-size:24px"></i>
```

Comment:
- [ ] Inline styling in your HTML is a bad practice, please use your CSS to style these elements 
![image](https://user-images.githubusercontent.com/53256139/93388870-fcb41380-f830-11ea-873a-3ca2390cd5c0.png)